addon/components/polaris-button/link.js

```diff
@@ -4,26 +4,18 @@ import mapEventToAction from '../../utils/map-event-to-action';
 
 export default BaseComponent.extend({
   tagName: 'a',
-  attributeBindings: [
-    'url:href',
-    'dataPolarisUnstyled:data-polaris-unstyled',
-    'target',
-    'rel',
-  ],
+  attributeBindings: ['url:href', 'dataPolarisUnstyled:data-polaris-unstyled', 'target', 'rel'],
```

Comment:
Not keen on this getting single-lined, there are quite a few attributes in this list...

Reply:
What's the line length limit before these get wrapped onto individual lines? This whole line is approaching 100 characters and seems like it should be getting split onto separate lines...

Reply:
100 chars. Should we make it 80?

Reply:
Yeah that might be a good idea... 100 seems kinda long